examples/variablesmatching.cf

```diff
@@ -44,5 +44,6 @@ bundle agent run
 #@ R: Variables matching 'default:sys.cf_version.*' = default:sys.cf_version_major
 #@ R: Variables matching 'default:sys.cf_version.*' = default:sys.cf_version_minor
 #@ R: Variables matching 'default:sys.cf_version.*' = default:sys.cf_version_patch
+#@ R: Variables matching "default:sys.cf_version.*" = default:sys.cf_version_release
```

Comment:
@vpodzime need to use single quotes here.

Reply:
Yeah, saw that :disappointed:

Reply:
The crazy thing is the output from the tests, though:
```
Q: "...r/bin/perl /hom": -R: Variables matching "default:sys.cf_version.*" = default:sys.cf_version_release
Q: "...r/bin/perl /hom": +R: Variables matching "default:sys.cf_version.*" = default:sys.cf_version_release
```
That's where I took the double-quotes from.

Reply:
@vpodzime it seems cf-agent replaces the quotes from the diff output?